venture-studio.html

```diff
@@ -76,7 +77,7 @@ <h1 class="to-animate">dltx.io</h1>
         <div class="row">
           <!-- HERO SECTION PHILOSOPHY -->
           <div class="col-md-12 section-heading text-center">
-            <h2 class="to-animate">Venture Studio</h2>
+            <h2 class="to-animate">The Studio</h2>
```

Comment:
Did you mean to change that?
```suggestion
            <h2 class="to-animate">Venture Studio</h2>
```